deploy/.env.example

```
@@ -0,0 +1,10 @@
+DB_USERNAME=my4cut
+DB_PASSWORD=replace-me
+JWT_SECRET_KEY=replace-me
+RESEND_API_KEY=replace-me
+MAIL_FROM=MY4CUT <noreply@example.com>
+
+# Production push notifications are enabled by default.
+# Keep the service-account JSON outside the repository and restrict its file permissions.
+FIREBASE_ENABLED=true
+FIREBASE_CONFIG_FILE=./secrets/firebase-service-account.json
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The FIREBASE_CONFIG_FILE key should go before the FIREBASE_ENABLED key

(UnorderedKey)